hack/env.sh

```diff
@@ -1,9 +1,12 @@
-export SRIOV_CNI_IMAGE=quay.io/openshift/origin-sriov-cni:4.2.0
-export SRIOV_DEVICE_PLUGIN_IMAGE=quay.io/openshift/origin-sriov-network-device-plugin:4.2.0
-export NETWORK_RESOURCES_INJECTOR_IMAGE=quay.io/openshift/origin-sriov-dp-admission-controller:4.2.0
+CNI_IMAGE_DIGEST=$(skopeo inspect docker://quay.io/openshift/origin-sriov-cni | jq --raw-output '.Digest')
```

Comment:
nit: This requires `skopeo` and `jq` be installed first, may be worth documenting.